clcache.py

```diff
@@ -1507,10 +1507,7 @@ def main():
     if "CLCACHE_DISABLE" in os.environ:
         return invokeRealCompiler(compiler, sys.argv[1:])[0]
     try:
-        exitCode, compilerStdout, compilerStderr = processCompileRequest(cache, compiler, sys.argv)
-        printBinary(sys.stdout, compilerStdout.encode(CL_DEFAULT_CODEC))
-        printBinary(sys.stderr, compilerStderr.encode(CL_DEFAULT_CODEC))
-        return exitCode
+        return processCompileRequest(cache, compiler, sys.argv)
```

Comment:
I've placed this in a separate method. It's needed since we don't want to print the aggregated output of all compiler invokations in one go but want to print information as soon as it is available (i.e., we have processed the cache hit or the compiler invokation has returned).